src/features/events_list/components/EventsListPanel/EventsListPanel.module.css

```diff
@@ -1,8 +1,9 @@
 .panelContainer {
   display: flex;
   flex-flow: column nowrap;
-  &.open {
-    height: 100%;
+  &.short .contentWrap {
```

Comment:
it looks like you don't need this override and .short class at all 
you can use Panel prop
        minContentHeightPx={isShortStateOpen ? 'min-content' : undefined} 
 it just get only number, maybe it makes sense to minContentHeightPx -> minContentHeight and get string too 🤔

Reply:
unfortunately i need also to reset inline property of 'height' for content container with this ugly `!important`

Reply:
i've just removed this block and can't find short state broken or something
could you please clarify how we need it?

Reply:
without it short state would be higher than its content
did you pull my latest changes?

Reply:
yeah, found it. 
maybe it makes sense to do in throught panel props? 
we already have minContentHeight, so maybe it's a good reason to introduce contentHeight prop? so no need overrides in css then
what do you think @Akiyamka ? does it worth it, or we'll go with current approach?

Reply:
I'll try to not forget to add it, sounds reasonable